scripts/quests/windurst/Curses_Foiled_A_Golem.lua

```diff
@@ -185,6 +185,9 @@ quest.sections =
             onEventFinish =
             {
                 [342] = function(player, csid, option, npc)
+                    -- Per FFXIclopedia: You can still get [Total Loser] at the
+                    -- title-changing NPC after the quest has been completed.
+                    player:addTitle(xi.title.TOTAL_LOSER)
```

Comment:
If the quest awards 2 titles on completion, should we modify the framework so that multiple titles can be given in the rewards block?

Reply:
I can't think of many other occasions where this would be needed; however, after looking at the addTitle binding, it looks like the function was changed from its original purpose and adds _and sets_ the player's title, which means we're going to be sending two update packets back to back here unless a change is made